src/hn/ui/CommentItem.ui

```diff
@@ -24,25 +22,26 @@
         </child>
         <child>
           <object class="GtkLabel" id="comment">
-            <property name="visible">True</property>
-            <property name="can_focus">False</property>
-            <property name="vexpand">True</property>
-            <property name="hexpand">True</property>
+            
+            <property name="can_focus">0</property>
+            <property name="vexpand">1</property>
+            <property name="hexpand">1</property>
             <property name="label" translatable="yes">...</property>
-            <property name="wrap">True</property>
-            <property name="selectable">True</property>
+            <property name="wrap">1</property>
+            <property name="selectable">1</property>
             <property name="xalign">0</property>
             <signal name="activate-link" handler="activate_link" swapped="no"/>
           </object>
         </child>
         <child>
           <object class="GtkButton" id="revealer_event">
+            
             <signal name="clicked" handler="reveal_replies_click" swapped="no"/>
             <child>
               <object class="GtkImage" id="revealer_img">
-                <property name="visible">True</property>
-                <property name="can_focus">False</property>
-		<property name="resource">/hn/icons/arrow_up.svg</property>
+                
+                <property name="can_focus">0</property>
+                <property name="resource">/hn/icons/arrow_up.svg</property>
```

Comment:
If you're using gresources and set up the icon dirs properly, you can use the `icon-name` property of GtkButton